examples/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_ls8cls9c_gm_cyear_3.js

```diff
@@ -0,0 +1,27 @@
+var geomedian_ls8ls9 = ee.ImageCollection('projects/geoscience-aus-cat/assets/ga_ls8cls9c_gm_cyear_3');
```

Comment:
could you please make the collection publicly readable?

Reply:
Readable now: https://code.earthengine.google.com/?asset=projects/geoscience-aus-cat/assets/ga_ls8cls9c_gm_cyear_3